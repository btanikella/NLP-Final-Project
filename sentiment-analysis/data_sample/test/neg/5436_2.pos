DT NN  IN RB RB JJ  VBD JJ  CC DT NN RB VBZ PRP
DT NN IN DT NNS IN DT NN VBP JJ IN DT NN CC DT NN  CC NNP NNP POS NN VBZ IN DT RB VBN NN WP VBZ IN NN CC VBZ RP VBG DT JJ NN IN PRP$ NN TO DT JJ NN IN NN WP MD RB VB NN IN PRP RB VBZ DT JJS NN IN PRP MD VB JJR IN JJ
WRB NNS VBP DT NN WDT RB CC VB DT JJ NN  PRP MD VB DT NN
