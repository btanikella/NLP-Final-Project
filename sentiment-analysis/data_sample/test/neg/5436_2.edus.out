Sentence #1 (18 tokens):
The book, while not particularly great, was decent, but this movie completely changes it.
[Text=The CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=DT] [Text=book CharacterOffsetBegin=8 CharacterOffsetEnd=12 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=12 CharacterOffsetEnd=13 PartOfSpeech=,] [Text=while CharacterOffsetBegin=14 CharacterOffsetEnd=19 PartOfSpeech=IN] [Text=not CharacterOffsetBegin=20 CharacterOffsetEnd=23 PartOfSpeech=RB] [Text=particularly CharacterOffsetBegin=24 CharacterOffsetEnd=36 PartOfSpeech=RB] [Text=great CharacterOffsetBegin=37 CharacterOffsetEnd=42 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=42 CharacterOffsetEnd=43 PartOfSpeech=,] [Text=was CharacterOffsetBegin=44 CharacterOffsetEnd=47 PartOfSpeech=VBD] [Text=decent CharacterOffsetBegin=48 CharacterOffsetEnd=54 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=54 CharacterOffsetEnd=55 PartOfSpeech=,] [Text=but CharacterOffsetBegin=56 CharacterOffsetEnd=59 PartOfSpeech=CC] [Text=this CharacterOffsetBegin=60 CharacterOffsetEnd=64 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=65 CharacterOffsetEnd=70 PartOfSpeech=NN] [Text=completely CharacterOffsetBegin=71 CharacterOffsetEnd=81 PartOfSpeech=RB] [Text=changes CharacterOffsetBegin=82 CharacterOffsetEnd=89 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=90 CharacterOffsetEnd=92 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=92 CharacterOffsetEnd=93 PartOfSpeech=.] 
root(ROOT-0, decent-10)
det(book-2, The-1)
nsubj(decent-10, book-2)
punct(book-2, ,-3)
mark(great-7, while-4)
neg(great-7, not-5)
advmod(great-7, particularly-6)
dep(book-2, great-7)
punct(book-2, ,-8)
cop(decent-10, was-9)
punct(decent-10, ,-11)
cc(decent-10, but-12)
det(movie-14, this-13)
nsubj(changes-16, movie-14)
advmod(changes-16, completely-15)
conj(decent-10, changes-16)
dobj(changes-16, it-17)
punct(decent-10, .-18)

Sentence #2 (68 tokens):
A lot of the elements of the story are consistent between the book and the movie, but Dr. Ross' character goes from a creatively written character who lives for money and ends up causing the volcanic eruption with her greed to a heart-on-her-sleeve damsel in distress who won't do anything if she even catches the slightest hint that it might be less than noble.
[Text=A CharacterOffsetBegin=98 CharacterOffsetEnd=99 PartOfSpeech=DT] [Text=lot CharacterOffsetBegin=100 CharacterOffsetEnd=103 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=104 CharacterOffsetEnd=106 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=107 CharacterOffsetEnd=110 PartOfSpeech=DT] [Text=elements CharacterOffsetBegin=111 CharacterOffsetEnd=119 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=120 CharacterOffsetEnd=122 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=123 CharacterOffsetEnd=126 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=127 CharacterOffsetEnd=132 PartOfSpeech=NN] [Text=are CharacterOffsetBegin=133 CharacterOffsetEnd=136 PartOfSpeech=VBP] [Text=consistent CharacterOffsetBegin=137 CharacterOffsetEnd=147 PartOfSpeech=JJ] [Text=between CharacterOffsetBegin=148 CharacterOffsetEnd=155 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=156 CharacterOffsetEnd=159 PartOfSpeech=DT] [Text=book CharacterOffsetBegin=160 CharacterOffsetEnd=164 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=165 CharacterOffsetEnd=168 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=169 CharacterOffsetEnd=172 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=173 CharacterOffsetEnd=178 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=178 CharacterOffsetEnd=179 PartOfSpeech=,] [Text=but CharacterOffsetBegin=180 CharacterOffsetEnd=183 PartOfSpeech=CC] [Text=Dr. CharacterOffsetBegin=184 CharacterOffsetEnd=187 PartOfSpeech=NNP] [Text=Ross CharacterOffsetBegin=188 CharacterOffsetEnd=192 PartOfSpeech=NNP] [Text=' CharacterOffsetBegin=192 CharacterOffsetEnd=193 PartOfSpeech=POS] [Text=character CharacterOffsetBegin=194 CharacterOffsetEnd=203 PartOfSpeech=NN] [Text=goes CharacterOffsetBegin=204 CharacterOffsetEnd=208 PartOfSpeech=VBZ] [Text=from CharacterOffsetBegin=209 CharacterOffsetEnd=213 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=214 CharacterOffsetEnd=215 PartOfSpeech=DT] [Text=creatively CharacterOffsetBegin=216 CharacterOffsetEnd=226 PartOfSpeech=RB] [Text=written CharacterOffsetBegin=227 CharacterOffsetEnd=234 PartOfSpeech=VBN] [Text=character CharacterOffsetBegin=235 CharacterOffsetEnd=244 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=245 CharacterOffsetEnd=248 PartOfSpeech=WP] [Text=lives CharacterOffsetBegin=249 CharacterOffsetEnd=254 PartOfSpeech=VBZ] [Text=for CharacterOffsetBegin=255 CharacterOffsetEnd=258 PartOfSpeech=IN] [Text=money CharacterOffsetBegin=259 CharacterOffsetEnd=264 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=265 CharacterOffsetEnd=268 PartOfSpeech=CC] [Text=ends CharacterOffsetBegin=269 CharacterOffsetEnd=273 PartOfSpeech=VBZ] [Text=up CharacterOffsetBegin=274 CharacterOffsetEnd=276 PartOfSpeech=RP] [Text=causing CharacterOffsetBegin=277 CharacterOffsetEnd=284 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=285 CharacterOffsetEnd=288 PartOfSpeech=DT] [Text=volcanic CharacterOffsetBegin=289 CharacterOffsetEnd=297 PartOfSpeech=JJ] [Text=eruption CharacterOffsetBegin=298 CharacterOffsetEnd=306 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=307 CharacterOffsetEnd=311 PartOfSpeech=IN] [Text=her CharacterOffsetBegin=312 CharacterOffsetEnd=315 PartOfSpeech=PRP$] [Text=greed CharacterOffsetBegin=316 CharacterOffsetEnd=321 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=322 CharacterOffsetEnd=324 PartOfSpeech=TO] [Text=a CharacterOffsetBegin=325 CharacterOffsetEnd=326 PartOfSpeech=DT] [Text=heart-on-her-sleeve CharacterOffsetBegin=327 CharacterOffsetEnd=346 PartOfSpeech=JJ] [Text=damsel CharacterOffsetBegin=347 CharacterOffsetEnd=353 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=354 CharacterOffsetEnd=356 PartOfSpeech=IN] [Text=distress CharacterOffsetBegin=357 CharacterOffsetEnd=365 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=366 CharacterOffsetEnd=369 PartOfSpeech=WP] [Text=wo CharacterOffsetBegin=370 CharacterOffsetEnd=372 PartOfSpeech=MD] [Text=n't CharacterOffsetBegin=372 CharacterOffsetEnd=375 PartOfSpeech=RB] [Text=do CharacterOffsetBegin=376 CharacterOffsetEnd=378 PartOfSpeech=VB] [Text=anything CharacterOffsetBegin=379 CharacterOffsetEnd=387 PartOfSpeech=NN] [Text=if CharacterOffsetBegin=388 CharacterOffsetEnd=390 PartOfSpeech=IN] [Text=she CharacterOffsetBegin=391 CharacterOffsetEnd=394 PartOfSpeech=PRP] [Text=even CharacterOffsetBegin=395 CharacterOffsetEnd=399 PartOfSpeech=RB] [Text=catches CharacterOffsetBegin=400 CharacterOffsetEnd=407 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=408 CharacterOffsetEnd=411 PartOfSpeech=DT] [Text=slightest CharacterOffsetBegin=412 CharacterOffsetEnd=421 PartOfSpeech=JJS] [Text=hint CharacterOffsetBegin=422 CharacterOffsetEnd=426 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=427 CharacterOffsetEnd=431 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=432 CharacterOffsetEnd=434 PartOfSpeech=PRP] [Text=might CharacterOffsetBegin=435 CharacterOffsetEnd=440 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=441 CharacterOffsetEnd=443 PartOfSpeech=VB] [Text=less CharacterOffsetBegin=444 CharacterOffsetEnd=448 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=449 CharacterOffsetEnd=453 PartOfSpeech=IN] [Text=noble CharacterOffsetBegin=454 CharacterOffsetEnd=459 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=459 CharacterOffsetEnd=460 PartOfSpeech=.] 
root(ROOT-0, consistent-10)
det(lot-2, A-1)
nsubj(consistent-10, lot-2)
prep(lot-2, of-3)
det(elements-5, the-4)
pobj(of-3, elements-5)
prep(elements-5, of-6)
det(story-8, the-7)
pobj(of-6, story-8)
cop(consistent-10, are-9)
prep(consistent-10, between-11)
det(book-13, the-12)
pobj(between-11, book-13)
cc(book-13, and-14)
det(movie-16, the-15)
conj(book-13, movie-16)
punct(consistent-10, ,-17)
cc(consistent-10, but-18)
nn(Ross-20, Dr.-19)
poss(character-22, Ross-20)
possessive(Ross-20, '-21)
nsubj(goes-23, character-22)
conj(consistent-10, goes-23)
prep(goes-23, from-24)
det(character-28, a-25)
amod(character-28, creatively-26)
amod(character-28, written-27)
pobj(from-24, character-28)
nsubj(lives-30, who-29)
rcmod(character-28, lives-30)
prep(lives-30, for-31)
pobj(for-31, money-32)
cc(lives-30, and-33)
conj(lives-30, ends-34)
prt(ends-34, up-35)
xcomp(ends-34, causing-36)
det(eruption-39, the-37)
amod(eruption-39, volcanic-38)
dobj(causing-36, eruption-39)
prep(causing-36, with-40)
poss(greed-42, her-41)
pobj(with-40, greed-42)
prep(causing-36, to-43)
det(damsel-46, a-44)
amod(damsel-46, heart-on-her-sleeve-45)
pobj(to-43, damsel-46)
prep(damsel-46, in-47)
pobj(in-47, distress-48)
nsubj(do-52, who-49)
aux(do-52, wo-50)
neg(do-52, n't-51)
rcmod(damsel-46, do-52)
dobj(do-52, anything-53)
mark(catches-57, if-54)
nsubj(catches-57, she-55)
advmod(catches-57, even-56)
advcl(do-52, catches-57)
det(hint-60, the-58)
amod(hint-60, slightest-59)
dobj(catches-57, hint-60)
mark(less-65, that-61)
nsubj(less-65, it-62)
aux(less-65, might-63)
cop(less-65, be-64)
ccomp(hint-60, less-65)
prep(less-65, than-66)
pobj(than-66, noble-67)
punct(consistent-10, .-68)

Sentence #3 (19 tokens):
When movies change the book that much but keep the same name, it should be a crime.
[Text=When CharacterOffsetBegin=465 CharacterOffsetEnd=469 PartOfSpeech=WRB] [Text=movies CharacterOffsetBegin=470 CharacterOffsetEnd=476 PartOfSpeech=NNS] [Text=change CharacterOffsetBegin=477 CharacterOffsetEnd=483 PartOfSpeech=VBP] [Text=the CharacterOffsetBegin=484 CharacterOffsetEnd=487 PartOfSpeech=DT] [Text=book CharacterOffsetBegin=488 CharacterOffsetEnd=492 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=493 CharacterOffsetEnd=497 PartOfSpeech=WDT] [Text=much CharacterOffsetBegin=498 CharacterOffsetEnd=502 PartOfSpeech=RB] [Text=but CharacterOffsetBegin=503 CharacterOffsetEnd=506 PartOfSpeech=CC] [Text=keep CharacterOffsetBegin=507 CharacterOffsetEnd=511 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=512 CharacterOffsetEnd=515 PartOfSpeech=DT] [Text=same CharacterOffsetBegin=516 CharacterOffsetEnd=520 PartOfSpeech=JJ] [Text=name CharacterOffsetBegin=521 CharacterOffsetEnd=525 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=525 CharacterOffsetEnd=526 PartOfSpeech=,] [Text=it CharacterOffsetBegin=527 CharacterOffsetEnd=529 PartOfSpeech=PRP] [Text=should CharacterOffsetBegin=530 CharacterOffsetEnd=536 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=537 CharacterOffsetEnd=539 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=540 CharacterOffsetEnd=541 PartOfSpeech=DT] [Text=crime CharacterOffsetBegin=542 CharacterOffsetEnd=547 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=547 CharacterOffsetEnd=548 PartOfSpeech=.] 
root(ROOT-0, change-3)
advmod(change-3, When-1)
nsubj(change-3, movies-2)
det(book-5, the-4)
dobj(change-3, book-5)
det(crime-18, that-6)
advmod(crime-18, much-7)
cc(much-7, but-8)
conj(much-7, keep-9)
det(name-12, the-10)
amod(name-12, same-11)
dep(much-7, name-12)
punct(crime-18, ,-13)
nsubj(crime-18, it-14)
aux(crime-18, should-15)
cop(crime-18, be-16)
det(crime-18, a-17)
dep(book-5, crime-18)
punct(change-3, .-19)

